spring-cloud-stream-binder-kafka/src/main/resources/META-INF/spring.binders

```diff
@@ -1,2 +1,2 @@
 kafka:\
-org.springframework.cloud.stream.binder.kafka.config.KafkaBinderConfiguration
+org.springframework.cloud.stream.binder.kafka.config.KafkaBinderConfiguration
```

Comment:
Would be great to have a new line in the end of file.
The IDE should be configured for this trick.
And in addition: with that option we even wouldn't have this file for review here at all...